README.md

```diff
@@ -9,10 +9,18 @@ Tikui is a MIT-licensed free software project allowing you to create a web patte
 
 ## Prerequisites
 
-* [Node.js](https://nodejs.org)
+* [Node.js](https://nodejs.org) LTS version
 
 ## Development
 
+### Install dependencies
+
+After clone the repository, go in the root tikui directory and launch this command:
```

Comment:
`After the repository clone`

Reply:
Or maybe `After cloning the repository, please go to the root Tikui directory and run this command:` ?

Reply:
done